mitodl/micromasters
# Added MicroMasters-generated course certificates for FA courses

#### What are the relevant tickets?
Closes #2778 

#### What's this PR do?
Adds a dynamically generated HTML certificate for users that have passed a course

#### How should this be manually tested?
- Create a passing `FinalGrade` for some user in a course run
- Create a `MicromastersCertificate` for that `FinalGrade` (the hash value will be auto-generated if not specified)
- Make sure that you have a program page in the CMS for the program that the final grade's course run is a part of.
- In the CMS, navigate to the program page for that program, and add one or more "COURSE CERTIFICATE SIGNATORIES" (at the bottom; make sure you refer to the correct `Course`)
- Navigate to the certificate URL for the `MicromastersCertificate` that you created (`/certificates/<certificate_hash_value`)

#### Where should the reviewer start?
`certificates/views.py`

#### Any background context you want to provide?
- The title of the referenced issue (and some of the description) is outdated. The implementation for these certificates has nothing to do with the `edx-certificates` repo
- Process for generated the certificate hashes for users that have passed a course will be implemented in a separate issue

#### Screenshots (if appropriate)
![mm_course_cert](https://user-images.githubusercontent.com/14932219/29222816-1788da9a-7e92-11e7-9b92-7cf3be0888eb.png)
